src/action.ts

```diff
@@ -1,34 +1,35 @@
-import * as core from "@actions/core";
-import { context as contextType } from "@actions/github";
+import * as core from '@actions/core';
```

Comment:
How was this passing linting before? Is the linting not run as part of CI?

Reply:
I was wondering the same thing. It looks like `npm run lint` is part of the tests but that wouldn't fix formatting.